src/le.rs

```diff
@@ -4,7 +4,7 @@ use std::fmt;
 use std::num::Wrapping;
```

Comment:
We've got lots of warnings for unused imports! 😱

Reply:
I just discovered `cargo fix` :star_struck: